readthedocs/doc_builder/python_environments.py

```diff
@@ -193,12 +193,12 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.5',
+            'sphinx==1.5.3',
```

Comment:
@willingc good point, and actually looking at [a search result](https://github.com/sphinx-doc/sphinx/issues?utf8=%E2%9C%93&q=is%3Aissue%201.5) on the Sphinx issue tracker reveals that their 1.5 series does not seem to carry many issues specific to patch versions, so it's low risk.

[This issue](https://github.com/sphinx-doc/sphinx/issues/3367) in Sphinx 1.5.2 might be an example of how RTD would have to skip a patch release due to an issue, however that could be fixed reactively when necessary.